sdk/core/azure-core/src/main/java/com/azure/core/http/policy/UserAgentPolicy.java

```diff
@@ -23,6 +23,10 @@
     // azsdk-java-<client_lib>/<sdk_version> <platform_info>
     private static final String USER_AGENT_FORMAT = DEFAULT_USER_AGENT_HEADER + "-%s/%s %s";
 
+    // From the design guidelines, the application id user agent header format is:
```

Comment:
The link in the JavaDoc for the class is missing its closing " at the end of the href. Can you please fix that.

Reply:
updated